utils/ci/run_in_gha.sh

```diff
@@ -80,6 +80,27 @@
 ./utils/setup_daos_server_helper.sh
 echo ::endgroup::
 
+echo ::group::Tar /opt/daos et /home/daos/daos
+tar czf /tmp/$DOCKER_BASE_opt_daos.tgz /opt/daos
+tar czf /tmp/$DOCKER_BASE_home_daos.tgz /home/daos
```

Comment:
(lint) DOCKER_BASE_home_daos is referenced but not assigned. [SC2154]
(lint) Double quote to prevent globbing and word splitting. [SC2086]